XRTK-Core/Packages/com.xrtk.core/Inspectors/Profiles/MixedRealityServiceProviderProfileInspector.cs

```diff
@@ -30,7 +31,7 @@ protected override void OnEnable()
 
             Debug.Assert(configurations != null);
             var baseType = ThisProfile.GetType().BaseType;
-            var genericTypeArgs = baseType?.GenericTypeArguments;
+            var genericTypeArgs = baseType.FindTopmostGenericTypeArguments();
```

Comment:
Base type could still be null?

```suggestion
            var genericTypeArgs = baseType?.FindTopmostGenericTypeArguments();
```